p2p/libp2p/defineOptions.go

```diff
@@ -0,0 +1,49 @@
+package libp2p
```

Comment:
The file name is too generic, make it manager specific

Reply:
I follow the same pattern found in the whole project. Won't change.

Reply:
there is only one other file with this name